src/tpm2_wrap.c

```diff
@@ -1726,6 +1843,30 @@ int wolfTPM2_RsaKey_WolfToTpm(WOLFTPM2_DEV* dev, RsaKey* wolfKey,
 {
     return wolfTPM2_RsaKey_WolfToTpm_ex(dev, NULL, wolfKey, tpmKey);
 }
+
+int wolfTPM2_RsaKey_PemToTpm(WOLFTPM2_DEV* dev, WOLFTPM2_KEY* tpmKey,
+                             byte *tempBuf, int tempSz, const char *filename)
```

Comment:
I would prefer a buffer PEM input, not filename. Use separate code to write to file. Use `wc_PubKeyPemToDer`

Reply:
My intention was to create a function that allows a user to import Private PEM Key into the TPM using this helper. I imagined a scenario, where a user wants to import his OpenSSH or OpenVPN key.

I think wc_PemPubKeyToDer allows conversion of both private and public keys, correct?

I made the change you requested. Perhaps I would have to rename the function to wolfTPM2_Rsa_Key_PubPemToTpm, so we leave space for future API of importing private PEM ?